.github/ISSUE_TEMPLATE/upgrade-liberty-plugins.md

```diff
@@ -8,8 +8,6 @@ assignees: ''
 ---
 # Upgrade to the latest Liberty Maven and Gradle Plug-ins Releases
 
-***Make sure to also update the `Add to an existing application` snippets on the start page via the [website code](https://github.com/OpenLiberty/openliberty.io)**
```

Comment:
This looks to be unrelated to the changes in `README.txt` and are fine with staying in the code base.